java_client_src/src/ihs/apcs/spacebattle/GraphicalClient.java

```diff
@@ -72,7 +72,7 @@ public void run() {
 			
 			// Create ship
 			this.shipClassname = args[1];
-			Class<?> shipType = Class.forName(this.shipClassname);
+			shipType = Class.forName(this.shipClassname);
```

Comment:
```suggestion
			this.shipType = Class.forName(this.shipClassname);
```

Should use `this.` to be consistent with the rest of the conventions here.